pkgs/development/python-modules/pymediainfo/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchPypi, buildPythonPackage
+, libmediainfo
+, pytest, glibcLocales }:
+
+buildPythonPackage rec {
+  pname = "pymediainfo";
+  version = "2.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "160yyjyfg441h30pqfr6370hcch17j3s0x2f1l0r220xgy969mr1";
+  };
+
+  postPatch = ''
+    substituteInPlace pymediainfo/__init__.py \
+      --replace 'CDLL("libmediainfo.so.0")' 'CDLL("${libmediainfo}/lib/libmediainfo.so.0")' \
```

Comment:
`${stdenv.hostPlatform.extensions.sharedLibrary}` instead of `.so`

Reply:
Done. I've also replaced the library references for Darwin.